.github/CONTRIBUTING.md

```diff
@@ -174,30 +176,46 @@ The config for a pre-commit hook is stored in [.pre-commit-config](../.pre-commi
 
 #### Installing pre-commit hooks
 
-After you clone the repository, you will need to install initialize pre-commit hook.
+##### Prerequisite
 
-```shell
-pip install -U pre-commit
-```
+Make sure Ruby runs on your system.
 
-From the repository folder
+On Windows: Install Ruby from [the official website](https://rubyinstaller.org/).
+
+On Debian/Ubuntu:
 
 ```shell
-pre-commit install
+sudo apt-add-repository ppa:brightbox/ruby-ng -y
+sudo apt-get update
+sudo apt-get install -y ruby2.7
 ```
 
-If you are facing issue when installing markdown lint, you may install ruby for markdown lint by following
+On other platforms:
```

Comment:
'other platforms' might not be clear enough? macOS? or other Linux distributions? or what?